Readme.md

```diff
@@ -16,9 +16,11 @@
 If you get it working on Windows, you should make an instruction file and
 submit a pull request!
 
-1. Clone or unzip this repository into your chosen directory
-2. Open a terminal to said directory and type `npm install` (this will create a *large* `node_modules` directory)
-3. Type `gulp` - the app should open in your default browser
++1. Clone or unzip this repository into your chosen directory
++2. Open a terminal to said directory and type `npm install` (this will create a *large* `node_modules` directory)
++2.5. (Windows only) In the terminal window type 'npm install -g gulp'. This allows the CLI to access gulp
++3. Add the URL to your firebase account "var base = Rebase.createClass('YOUR URL HERE');" in ListManager.js
```

Comment:
Two things here:

1. Wrap the code (`var base = ...`) in backticks for nice formatting. (Underneath the tilde, left of the 1 key (on a Mac, anyway).)

2. For the uninitiated, specify the full path to `ListManager.js`: `js/components/ListManager.js`